src/index.ts

```diff
@@ -50,6 +50,19 @@ type FieldComponent<Values extends Record<string, any>, ErrorMessage = string> =
   displayName?: string;
 };
 
+type FieldsListenerComponent<Values extends Record<string, any>, ErrorMessage = string> = (<
```

Comment:
maybe we should use `unknown` instead of `any`. Or it will bring a TS error?

Reply:
Actually it doesn't really matter here since it's used in `extends`. But I could change it to `unknown` yes.